server/app/routes.py

```diff
@@ -13,3 +20,44 @@ def videos_raw():
 	@app.route('/videos')
 	def videos():
 		return jsonify(rss.summarize_watchlist(app.config['watchlist']))
+
+	@app.route('/watchlist', methods=['POST', 'DELETE'])
+	async def modify_watchlist():
+		# only required value is 'id'
+		json = request.get_json()
+		if 'id' not in json and 'url' not in json:
+			return '', 400 # bad request
+		if 'id' in json and 'url' in json:
+			return '', 400 # bad request
```

Comment:
this would be a good case where we can return a descriptive error message in addition to 400

Reply:
Done